jfrog_platform_cloud/README.md

```diff
@@ -0,0 +1,30 @@
+# Agent Check: JFrog Platform Cloud
+
+## Overview
+
+[JFrog][1] is a universal hybrid, end-to-end DevOps platform. This integration allows any JFrog cloud customer to seamlessly stream logs from JFrog Artifactory into Datadog. This integration provides Datadog [log pipelines][2] which enrich and index logs to make them more searchable and treatable using Datadog [facets][3].
+
+### JFrog dashboards
+
+You can find the dashboard packaged with this integration under the Assets tab on the integration tile.
+
+#### JFrog Platform Cloud Log Analytics
+This dashboard provides a summary of access logs and information about various HTTP response codes, HTTP 500 errors, call volume, and user operations.
+
+## Setup
+
+### Installation
+
+Contact a JFrog representative to enable the integration. Be ready to provide your [Datadog API key][4].
```

Comment:
Can you link who they should be reaching out to?

Reply:
Added.